src/tests/dictionaryConcepts/components/CreateConceptForm.test.jsx

```diff
@@ -173,4 +191,62 @@ describe('Test suite for CreateConceptForm', () => {
     expect(wrapper.find('CreateMapping').length).toBe(1);
     wrapper.unmount();
   });
+
+  it('should render form for numericPrecision when dataType is numeric when creating concept', () => {
+    const props = {
+      state: {
+        id: '1',
+      },
+      concept: 'oldOne',
+      addDescription: jest.fn(),
+      handleNewName: jest.fn(),
+      path: '',
+      toggleUUID: jest.fn(),
+      handleChange: jest.fn(),
+      handleSubmit: jest.fn(),
+      editable: false,
+      addAnswer: jest.fn(),
+      answer: [],
+      nameRows: [],
+      description: [],
+      isEditConcept: false,
+      disableButton: false,
+      numericPrecisionOptions:{
+        numericEnabled: true,
+      },
+      allSources: [mockSource],
+    };
+    const wrapper = shallow(<CreateConceptForm {...props} />);
+    expect(wrapper.find('NumericPrecision')).toHaveLength(1);
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it('should render form for numericPrecision when dataType is numeric', () => {
```

Comment:
What is the difference between this test and the one above?

Reply:
I will clarify the test name...This is intended to test when your editing a concept and not creating......(line 212 and line 241)